server/neptune/workflows/internal/activities/terraform.go

```diff
@@ -59,28 +66,49 @@ func (t *terraformActivities) TerraformInit(ctx context.Context, request Terrafo
 	}
 
 	ch := t.TerraformClient.RunCommand(ctx, request.JobID, request.Path, cmd.Build(), request.Envs, tfVersion)
-	var lines []string
-	for line := range ch {
-		if line.Err != nil {
-			err = errors.Wrap(line.Err, "executing command")
-			break
-		}
-		lines = append(lines, line.Line)
+	_, err = t.readTerraformOutput(ch)
+	if err != nil {
+		return TerraformInitResponse{}, errors.Wrap(err, "processing command output")
 	}
-	output := strings.Join(lines, "\n")
-
-	// sanitize output by stripping out any ansi characters.
-	output = ansi.Strip(output)
 	return TerraformInitResponse{}, nil
 }
 
 // Terraform Plan
-
 type TerraformPlanRequest struct {
```

Comment:
This struct is exactly the same as `TerraformInitRequest`. Thoughts on reusing the same struct? Maybe `TerraformRequest`?

Reply:
I'd rather keep them separate for now, just for clarity purposes/if things change over time.